insta_posts-scraper/posts-scraper.go

```diff
@@ -29,18 +29,17 @@ type InstaPostsScraper struct {
 	errQWriter   *kafka.Writer
 
 	requestCounter int
-
-	httpClient scraper_client.ScraperClient
+	httpClient     scraper_client.ScraperClient
 }
 
 // New returns an initilized scraper
-func New(nameQReader *kafka.Reader, infoQWriter *kafka.Writer, errQWriter *kafka.Writer) *InstaPostsScraper {
+func New(awsServiceAddress string, nameQReader *kafka.Reader, infoQWriter *kafka.Writer, errQWriter *kafka.Writer) *InstaPostsScraper {
 	i := &InstaPostsScraper{}
 	i.nameQReader = nameQReader
 	i.postsQWriter = infoQWriter
 	i.errQWriter = errQWriter
 
-	i.httpClient = scraper_client.NewSimpleScraperClient()
+	i.httpClient = scraper_client.NewHttpClient(awsServiceAddress)
```

Comment:
Can we make the usage of either `NewHttpClient` or `NewSimpleScraperClient` configurable? When we are not inside aws, we don't want to use `NewHttpClient `

Reply:
yeah sure, @jo-fr  already suggested that